packages/aws-rfdk/lib/deadline/scripts/python/configure_identity_registration_settings.py

```diff
@@ -173,10 +158,10 @@ def _secret(value):
         match = FILE_URI_RE.match( value )
         if match:
             named_groups = match.groupdict()
-            return FileSecret( arn=named_groups['FilePath'] )
+            return FileSecret( filepath=named_groups['FilePath'] )
```

Comment:
Was using a `FileSecret` tested end-to-end? Do we see any use case where we want this?

Reply:
You're right - we should remove it. This script was ported from our [`client-rq-connection.py`](https://github.com/aws/aws-rfdk/blob/8845f4787cd7761f4b94213a7577ad6745d4246e/packages/aws-rfdk/lib/deadline/scripts/python/client-rq-connection.py) and the `FileSecret` came along for the ride. We can remove it since we don't actually use it.